api/aether-top-level-openapi3.yaml

```diff
@@ -131,79 +131,302 @@ components:
       required:
         - target_id
       type: object
+    Failure:
+      properties:
+        type:
+          description: transaction failure type
+          type: string
+          enum:
+            - UNKNOWN
+            - CANCELED
+            - NOT_FOUND
+            - ALREADY_EXISTS
+            - UNAUTHORIZED
+            - FORBIDDEN
+            - CONFLICT
+            - INVALID
+            - UNAVAILABLE
+            - NOT_SUPPORTED
+            - TIMEOUT
+            - INTERNAL
+        description:
+          type: string
+    Revision:
+      type: integer
+      format: uint64
```

Comment:
Actually OpenAPI3 spec does not support `uint` of any type
https://spec.openapis.org/oas/v3.0.0#data-types. Just make it `int64`

Reply:
Done